src/sources/file.rs

```diff
@@ -185,6 +193,7 @@ pub struct FileConfig {
 
     /// An approximate limit on the amount of data read from a single file at a given time.
```

Comment:
The wording of approximate makes me think we should explain that a little more in this comment, since it is now user facing.

Reply:
Ah 👍. I guess it could be an improvement then. But defer to your discretion.

Reply:
I'm not super familiar but I think it's approximate because we don't cap on an exact number of bytes - so it's a little fudgey regardless